README.md

```diff
@@ -109,11 +78,70 @@ s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler.template
 s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler-remote.template
 ```
 
-## Deploy
+### Deploy
 
 See the [AWS Instance Scheduler Implementation Guide](https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/instance-scheduler.pdf) for deployment instructions, using the link to the instance-scheduler.template from your bucket, rather than the one for AWS Solutions. Ex. https://mybucket.s3.amazonaws.com/aws-instance-scheduler/v1.4.0.mybuild/instance-scheduler.template
 
-## CDK Documentation
+
+# Testing the Solution
```

Comment:
Making this a second-level heading would cascade into the individual step's being fourth-level which does not look right to me

Reply:
`<h1>` normally corresponds to a page's title, and there's normally only one. I don't see this as a good reason to break that pattern. If the lower level elements don't look good when formatted, then you should figure out a better way to format them, rather than introducing a section that is at the same level as the title of the document.